lib/transaction.rb

```diff
@@ -0,0 +1,19 @@
+require 'date'
+
+class Transaction
+
+  attr_reader :date, :balance, :credit, :debit
+
+  def initialize(balance, credit = nil, debit = nil)
+    @date = formatted_date
```

Comment:
Is this the right moment to format the date? When will the date need to be formatted, and is that moment right now?